src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatString = sourceString.replace(/;\s*;/g, ' ').trim();
+
+  const stringToArray = formatString.split(';');
```

Comment:
[CODE STYLE] - The variable name `stringToArray` is not very descriptive. Consider renaming it to something more indicative of its contents, such as `declarations` or `styleRules`.